### ISSUE TITLE
Changes from Release Branch to Master

### ISSUE THREAD

**Issue by rstoermer** [MicrosoftDocs/azure-devops-docs#9304]
In this document you describe, bringing changes from the Release branch back to the mainline. However on https://docs.microsoft.com/en-us/azure/devops/learn/devops-at-microsoft/release-flow you say, which makes a lot of sense in my opinion:

More importantly, by following this process, we're guaranteed that our change goes into master. This is critical for us: if we were to fix a bug in the release branch first, and accidentally forget to bring the change back to master, we would have a recurrence of the bug during the next deploy - when we create our sprint 130 release branch from master in three weeks.

What is correct?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: fe168fc3-2ed2-b3e6-ce30-cac721165ceb
* Version Independent ID: 8d63fbd7-12c4-9a94-92f6-969e7eb41037
* Content: [Git branching guidance - Azure Repos](https://docs.microsoft.com/en-us/azure/devops/repos/git/git-branching-guidance?view=azure-devops)
* Content Source: [docs/repos/git/git-branching-guidance.md](https://github.com/MicrosoftDocs/azure-devops-docs/blob/master/docs/repos/git/git-branching-guidance.md)
* Product: **devops**
* Technology: **devops-code-git**
* GitHub Login: @vtbassmatt
* Microsoft Alias: **macoope**

**Comment by vtbassmatt** [comment]
Any time we offer concrete guidance, I guess we run the risk of suggesting conflicting ideas. Both approaches are valid; at the scale our team was operating (when we wrote those topics initially) we needed to start in `master` and port to release branches. Today, with even more maturity, we might decide it's more important to fix the release branch first and port to `master`. I'll see if I can make that clear without turning the topics into a pile of disclaimers 😁
